x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/http/TemplateHttpResource.java

```diff
@@ -98,8 +104,18 @@ protected void doPublish(final RestClient client, final ActionListener<Boolean>
      *
      * @return Never {@code null}.
      */
+    @SuppressWarnings("unchecked")
```

Comment:
what is the call that requires suppressing this warning?

Reply:
Good catch, that was from an earlier attempt and is no longer needed. It is now removed.